src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -66,9 +65,7 @@ def _authentication_context_factory(authority, cache):
 
 CLOUD = get_active_cloud()
 
-logger.debug("Current active cloud '%s'", CLOUD.name)
-logger.debug(pformat(vars(CLOUD.endpoints)))
-logger.debug(pformat(vars(CLOUD.suffixes)))
+logger.debug(str(CLOUD))
```

Comment:
Unless the to string conversion includes readable explanation to what it is, I would suggesting print a more readable message like: 
``` PYTHON
logger.debug('Current cloud is {}'.format(str(CLOUD)))
```